student_explorer/settings.py

```diff
@@ -91,6 +93,8 @@
 
 FORCE_SCRIPT_NAME = config('DJANGO_FORCE_SCRIPT_NAME', default=None)
 
+SITE_ID = 1
```

Comment:
Why do we need to set site_id here?

Reply:
https://docs.djangoproject.com/en/3.2/ref/contrib/flatpages/#installation

Reply:
Got it :)

Can you please add this link as a comment to the file? Thanks!

Reply:
I can. 

I'll just note that there are lots of settings in this file, but we don't have links for all of them. If we were maintaining this long-term, I'd be concerned about the file getting harder to read with comments everywhere. If I had forgotten what this was for, I'd just search for it like this: https://docs.djangoproject.com/en/4.2/search/?q=site_id

I did link to this in the PR description, which means you'd find it if you followed git blames and such.

Reply:
Sure. I will close the comment then.

Reply:
Sorry, already made the change for you.